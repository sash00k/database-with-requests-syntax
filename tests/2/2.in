get records where course is set