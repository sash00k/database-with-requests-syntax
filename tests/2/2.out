Teacher Власов Г. В. on course Ан.геом. Works with 2 groups and 2 students.
Student Ударников В. П. from group 17121, ММФ. Also teaches Программирование-2 in 1 groups and mentors 0 students.
Teacher Кузнецова Л. И. on course Программирование. Works with 2 groups and 3 students.